### PR TITLE
Fix #18: Added MATLAB

### LINKED ISSUE #18: Add MATLAB
See `Android` in README.md to get an idea of the contents of the PR for this issue.

### PR DESCRIPTION
**Pull request by Subhangini**
## Explanation

Added MATLAB related data
fixes #18

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Ohter Tips / Hacks

### CODE REVIEW

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Add more data , and Matlab will come after Lua , not lightworks , so make the changes , and don't add additional lines,Also resolve merge conflicts

File: README.md [hunk 1]
```diff
@@ -1163,6 +1178,18 @@ You can find our Code of Conduct [here](https://github.com/CSwala/awesome-comput
 
 This project follows the [MIT License](https://choosealicense.com/licenses/mit/).
 
+## Contributors 🥰:star:
```
Comment (riturajjain2000) [inline README.md]: This is already there , why adding it again?

Comment (Subhangini) [reply README.md]: I am not adding again. That issue was also solved by me so whenever i create  any pull request in a particular Project all the changes done by me is being shown

Comment (riturajjain2000) [reply README.md]: Then after that PR was merged , you have not fetched the develop branch
Fetch the develop , then this should not happen

**Comment by Subhangini** [comment]
> Add more data , and Matlab will come after Lua , not lightworks , so make the changes , and don't add additional lines,Also resolve merge conflicts

ok

File: README.md (hunk 1 above)
Comment (Subhangini) [reply README.md]: ok, I got it

Comment (Subhangini) [reply README.md]: @riturajjain2000  actually i am new with git so unable to do fetching can you help?

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Why Kuernetes and lightworks is also there as additions?

**Comment by Subhangini** [comment]
> Why Kuernetes and lightworks is also there as additions?

I told you earlier that i am not good with git so i solved merge conflict on github and it came this way

**Comment by riturajjain2000** [comment]
> > Why Kuernetes and lightworks is also there as additions?
> 
> I told you earlier that i am not good with git so i solved merge conflict on github and it came this way

Then I'll suggest you to first clear the basics, then Make the PR again

**Comment by Subhangini** [comment]
I will close this PR and make a new one again
Do I need to add more info for Matlab

**Pull request closed without merging**